src/sidebars/sidebars.json

```diff
@@ -2120,6 +2120,22 @@
             "url": "/apps/zendesk-connector"
         }
     ],
+    "blog": [
```

Comment:
@andyvan-ph The blog sidebar can be updated here! This is ready to be updated. 😊

Reply:
Thanks. Have this in my schedule for tomorrow.